packages/salesforcedx-vscode-core/test/jest/commands/deployExecutor.test.ts

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
I added these tests as separate files, but the deploy and retrieve executors are still inside baseDeployRetrieve.ts in this PR.  I actually think this works well because extracting the executors made the PR much larger, and with this arrangement the tests are well organized but the PR doesn't blow up with the refactor to move the executors out.  LMK what you think.

Reply:
+1 for not blowing up the PR 👏  that big a refactor should probably go separately, and I'm not sure it's worth touching such core functionality if you were still able to accomplish enough testing here.

Reply:
+1 and I'll probably split it up after ken's PRs are in.